Fhi.HelseId.Swagger/WebApplicationExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Hosting;
+using System.Net.Mime;
+
+namespace Fhi.HelseId.Swagger;
+
+public static class WebApplicationExtensions
+{
+    public static RouteHandlerBuilder UseHelseIdTestTokenEndpoint(this WebApplication app)
+    {
+        if (app.Environment.IsDevelopment())
```

Comment:
IsDevelopment works only when environment name is "Development".  
This  is also a runtime check.  Perhaps it should be a compile check, so that when things go to production, it will not be compiled in.  
And, we can't use !IsProduction either because that is tied to name "Production", and in our case we use  a different name for production

Reply:
The IsDevelopment() means it will only run on environment with name "Development".  That is normally the local developer environment. The Test environments are normally named differently.